daemon/commit.go

```diff
@@ -127,6 +129,15 @@ func (daemon *Daemon) Commit(name string, c *backend.ContainerCommitConfig) (str
 	if err != nil {
 		return "", err
 	}
+	if container.IsDead() {
+		err := fmt.Errorf("You cannot commit container %s which is Dead", container.ID)
+		return "", apierrors.NewBadRequestError(err)
```

Comment:
This will need to be updated to use the conflict error type rather than `apierrors` which is now gone.